package.json

```diff
@@ -1,12 +1,11 @@
 {
   "scripts": {
-    "test": "xo && ava"
+    "test": "xo && ava",
+    "release": "webstore upload --source=extension --auto-publish"
```

Comment:
Re `--auto-publish`, when would you ever want to upload and not publish?  Maybe it would make sense to have a command for both upload and publish? I think that's the most common use-case and should require the least amount of config.